src/helpers/countAnswers.ts

```diff
@@ -0,0 +1,17 @@
+import { AnswerEnum } from '../AnswerEnum';
+import { IAnswer } from '../models';
+
+export const countAnswers = (answers: IAnswer[]) => {
```

Comment:
![image](https://user-images.githubusercontent.com/7914985/126147228-5af66e11-be32-4931-b812-1e6b5e2376e3.png)
Not sure about naming. Maybe it's better to call this function `isRoomCompleted` or `isStageCompleted`